src/cpp/CMakeLists.txt

```diff
@@ -19,6 +19,43 @@ target_include_directories(kittie
 	$<INSTALL_INTERFACE:${CMAKE_INSTALL_PREFIX}/include>
 )
 
-install(TARGETS kittie DESTINATION ${CMAKE_INSTALL_PREFIX}/lib)
-install(FILES ${CMAKE_CURRENT_BINARY_DIR}/kittie.h DESTINATION ${CMAKE_INSTALL_PREFIX}/include)
+include(GNUInstallDirs)
+set(INSTALL_CONFIGDIR ${CMAKE_INSTALL_LIBDIR}/cmake/kittie)
 
+install(TARGETS kittie 
+	EXPORT kittie-targets 
+	LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR})
+install(FILES ${CMAKE_CURRENT_BINARY_DIR}/kittie.h DESTINATION ${CMAKE_INSTALL_INCLUDEDIR})
+
+install(EXPORT kittie-targets
+    FILE
+        kittieTargets.cmake
+    NAMESPACE
+        kittie::
+    DESTINATION
+        ${INSTALL_CONFIGDIR}
+)
+
+include (CMakePackageConfigHelpers)
+write_basic_package_version_file(
```

Comment:
What does the version file do? I'm not sure if it's being used by the `effisConfig.cmake` that gets written, unless it's used by `include(CMakeFindDependencyMacro)`